opm/simulators/timestepping/SimulatorReport.cpp

```diff
@@ -62,134 +65,151 @@ namespace Opm
         total_linearizations += sr.total_linearizations;
         total_newton_iterations += sr.total_newton_iterations;
         total_linear_iterations += sr.total_linear_iterations;
+        global_time = sr.global_time;
```

Comment:
I suggest adding a comment here to explain why regular assignment is correct for `global_time`.